Tests/RegressionTests.cs

```diff
@@ -29,7 +29,7 @@ public void BasicTemplateAlgorithm()
                 {"Total Trades", "1"},
                 {"Average Win", "0%"},
                 {"Average Loss", "0%"},
-                {"Compounding Annual Return", "311.484%"},
+                {"Compounding Annual Return", "280.522%"},
```

Comment:
Interesting, I can run the tests locally and they pass. Hopefully there's not a day of week dependency in the tests (I've done it accidentally before). Maybe it's an input data issue?